fetch.py

```diff
@@ -11,8 +13,11 @@
 def get(day):
     print(f"Fetching {day}")
     day_dir = os.path.join(aoc_base_dir, str(day))
+    year = datetime.now(tz=ZoneInfo("America/New_York")).year
     for label, k in keys.items():
-        r = requests.get(f"https://adventofcode.com/2022/day/{day}/input", cookies=dict(session=k))
+        r = requests.get(
+            f"https://adventofcode.com/{year}/day/{day}/input", cookies=dict(session=k)
```

Comment:
unrelated to the PR but lol we are creating a new dictionary on every call here unnecessarily, theres much bigger fish before we get here though

Reply:
your telling python code that theres a "unnecessary operation" in a network request
like, you should probably look at the problem of a synchronous network request repeated multiple times, no?

Reply:
It's a fetch script in python that gets run from a systemd timer. I think we're OK. If anything, this script needs a `thread.sleep(5)` call thrown in to avoid abusing AOC servers.